File: acceptance/sulabhghimire/int_to_roman.dart
class Solution {
  List<int> values = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1];
  List<String> symbol = [
    "M",
    "CM",
    "D",
    "CD",
    "C",
    "XC",
    "L",
    "XL",
    "X",
    "IX",
    "V",
    "IV",
    "I"
  ];

  String intToRoman(int num) {
    String res = '';

    bool cond = true;

    while (cond) {
      cond = false;
      for (int val in values) {
        if (val <= num) {
          String index = symbol[values.indexOf(val)];
          res = res + index;
          num = num - val;
          cond = true;
          break;
        }
      }
    }

    return res;
  }
}

File: acceptance/arjun/inttoroman.py
class Solution:
    def intToRoman(self,num):
        roman = ""
        equivalent = [[1000, "M"], [900, "CM"], [500, "D"], [400, "CD"], [100, "C"], [90, "XC"], [50, "L"], [40, "XL"], [10, "X"], [9, "IX"], [5, "V"], [4, "IV"], [1, "I"]]
        for i in range(len(equivalent)):
                while num >= equivalent[i][0]:
                        roman +=equivalent[i][1]
                        num -= equivalent[i][0]
        return(roman)

